data_measurements/dataset_statistics.py

```diff
@@ -431,6 +381,7 @@ def load_or_prepare_labels(self, load_only=False):
         or else uses what's available in the cache.
         Currently supports Datasets with just one label column.
         """
+        print(self.dset)
```

Comment:
Remove print statement?

Reply:
In general, are we logging using `logger`?

Reply:
Done, and yes.